.ci/conformance/tools/elastic_net/patcher.py

```diff
@@ -0,0 +1,28 @@
+import re
+import sys
+
```

Comment:
Is it really good idea to have multiple instances  of scripts for each alg? 

Also - why we make separate scripts here and not using daal4py mechanisms?

Reply:
We use wrappers from daal4py, but modify them by inserting call counts.

Reply:
Might be it worth to place this code in to daal4py wrappers instead of inserting call counts on top of it? 
for example conformance mode can be enabled in daal4py wrappers as non defalt

Reply:
@napetrov Do you mean to add counters to the daal4py repository itself?

Reply:
yes